tests/test_modifiers.py

```diff
@@ -86,6 +82,37 @@ def test_inserting_an_empty_if_statement(self, create_instance: Modifiers):
         assert isinstance(value, Modifiers)
         assert any(stmt in modified for stmt in STATEMENTS for modified in value.modified_contents)
 
+    def test_reversing_booleans(self, create_instance: Modifiers):
+        value = create_instance.reverse_booleans()
+
+        assert isinstance(value, Modifiers)
+        assert ("True" not in value.modified_contents[1]) != ("False" not in value.modified_contents[1])
+        assert (value.modified_contents[1].count("True") == 2) != (value.modified_contents[1].count("False") == 2)
```

Comment:
Does this test for both cases? `True` and `False` in the code that get switched, or just one case?

Reply:
That modifier changes only one type of boolean per line so this can't be a case

Reply:
Can resolve